Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/hibernate/AuditRepositoryDAO.java

```diff
@@ -145,13 +147,13 @@ public List queryAuditRepositoryOnCriteria(String eUserId, String ePatientId, Da
 
             if (startDate != null && endDate != null) {
                 aCriteria.add(Expression.between("timeStamp", new Date(startDate.getTime()),
-                        new Date(endDate.getTime())));
+                    new Date(endDate.getTime())));
             } else if (startDate != null && endDate == null) {
                 aCriteria.add(Expression.ge("timeStamp", new Date(startDate.getTime())));
             }
             queryList = aCriteria.list();
-        } catch (Exception e) {
-            LOG.error("Exception in AuditLog.get() occured due to :" + e.getMessage());
+        } catch (HibernateException e) {
+            LOG.error("Exception in AuditLog.get() occured due to :" + e.getLocalizedMessage(), e);
```

Comment:
If you need to make another commit on this PR, can you fix this typo?  Should be "occurred".